docs/xonshrc.rst

```diff
@@ -47,3 +47,62 @@ The colors of the ``ls`` command may be hard to read in a dark terminal. If so,
 .. code-block:: xonshcon
 
     >>> $LS_COLORS='rs=0:di=01;36:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:'
+    
+Make json data directly pastable
+--------------------------------
+With this snippet, xonsh will understand json data such as ``{ "name": "Tyler", "active": false, "age": none }``.
+
+.. code-block:: xonshcon
+
+    >>> import builtins 
+    >>> builtins.true = True
+    >>> builtins.false = False
+    >>> builtins.null = None
```

Comment:
Of course it's gross, but it's simple and works. Have been using it for over a year now. Seeing it's in a snippet section and people are explicitly asked to submit their's, I don't think anyone would take it as "warranty implied".

Reply:
I am not against it. It is fun to see how flexible xonsh can be.

Reply:
I actually love it 😀

Reply:
If you find it gross, maybe we can suggest a context wrapper for this:
```py
with NativeJSON():
    data = { "name": "Tyler", "active": false, "age": null }
```

This would clean up builtins on context exit.

Reply:
Yeah, I am not against it... though this is a general Python hack.  Maybe this should be a xontrib?

Reply:
@con-f-use - can you maybe add a "use at your own risk" comment? I think we could merge this in with that

Reply:
@scopatz : done.